src/airtop/client.py

```diff
@@ -104,7 +121,22 @@ def __init__(
         timeout: typing.Optional[float] = None,
         follow_redirects: typing.Optional[bool] = True,
         httpx_client: typing.Optional[httpx.AsyncClient] = None,
+        debug: typing.Optional[bool] = False,
     ):
         super().__init__(api_key=api_key, base_url=base_url, environment=environment, timeout=timeout, follow_redirects=follow_redirects, httpx_client=httpx_client)
         self.windows = AsyncAirtopWindows(client_wrapper=self._client_wrapper)
         self.sessions = AsyncAirtopSessions(client_wrapper=self._client_wrapper)
+        self.debug = debug
+
+    def log(self, message: str):
+        if self.debug:
+            print(message)
+
+    def warn(self, message: str):
+        logging.warning(message)
+    
+    def error(self, message: str):
+        logging.error(message)
+
+    async def batch_operate(self, urls: List[BatchOperationUrl], operation: Callable[[BatchOperationInput], Awaitable[BatchOperationResponse]], config: Optional[BatchOperateConfig] = None) -> List[Any]:
```

Comment:
I am only adding this ability to the `AsyncAirtop` client.  I don't know that the concurrency management involved is even possible to run in a synchronous environment.  All attempts I made at this failed.

Reply:
I might pair with @kvindascr to see if we can get the synchronous version working.  But not sure that should block this going out anyway (we can release that as a follow-up)